doc/ref_impl/cntt-ri2/chapters/chapter01.rst

```diff
@@ -0,0 +1,139 @@
+5. Provide detailed design for automation of deployment and testing, provide continuous integration and delivery pipeline for the RI
+6. Gap analysis with required actions for existing ecosystem within adjacent community projects
+
+Relationship to other communities and workstreams
+-------------------------------------------------
+
+The main communities involved in driving requirements and development of this RI are:
+
+-  **Anuket RC2 workstream**
+
+   -  The **RC2 workstream** within Anuket takes requirements from the RM and RA2 and identifies test cases and a test integration suite that will be used to verify and validate conformance of an implementation. RI2 will be a deployable reference for operators, vendors, and others to develop against.
+
+-  **Anuket Kuberef**
+
+   -  The **Kuberef** project in Anuket acts as the main integration point for RI2: Its purpose is to provide all necessary glue code and mechanisms for i) integrating and configuring selected upstream Kubernetes deployers and components into a deployable platform, ii) continuously deploy the integrated platform in Anuket labs and iii) run the latest RC2 test suite against the deployed platform. The test results are meant to be fed back to RA2 and RC2 to support the refinement of architecture requirements and the selection of test cases for the RC2 test suite.
+
+-  **Anuket Functest**
+
+   -  The **Functest** project integrates available test tools and test cases from across the open source ecosystem into a single framework. In particular, it allows to run the test cases selected by the RC2 workstream and outputs a test result package for submission to the Anuket Assured program.
+
+-  **Compliance and Verification Committee & Cloud Native OVP aka OVP 2.0**
+
+   -  The Compliance and Verification Committee governs the Cloud Native OVP program which owns the definition of the end-to-end framework (tooling, process, tests, review, badging) used to verify conformance of a cloud native infrastructure implementation with the RA2 specifications.
+
+-  **CNCF - CNF Conformance, Kubernetes, etc.**
+
+   -  CNCF hosts a wide variety of projects providing software components for RI2, such as Kuberntes itself, deployment tooling (e.g. kubeadm and kubespray) as well as test suites (e.g., Kubernetes conformance).
+   -  The Telco User Group (TUG) is a forum within CNCF to identify and discuss telco requirements on the Kubernetes ecosystem.
+   -  The CNF Conformance project in CNCF has a stated aim to provide "visibility into how well Cloud native Network Functions (CNFs) and the underlying Telecom platform follows cloud native principles".
+   -  The CNF Testbed is an initiative providing a framework for building and deploying technology show cases with a strong focus on telco platform requirements.
+
+The following figure gives an overview of the relationship of the communities involved.
+
+|Relationship of communities|
```

Comment:
What about the Figure caption?

Reply:
added in the latest update. I am also using the rst reference mechanism to reference the figure (by name).